### ISSUE TITLE
NTR: Mevalonate lactone high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8638]
New term request Mevalonate lactone high in urine (elthat_190101150337):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000675
- label: Elevated urinary mevalonate lactone level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: PMID:29909962
- def: The amount of mevalonate lactone in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: This finding has been observed in squalene synthase deficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8638
- synonyms:
